src/ServicePulse.Host/vue/src/components/DataView.vue

```diff
@@ -13,6 +17,7 @@ const props = withDefaults(
     autoRefreshSeconds: number;
     showPagination?: boolean;
     showItemsPerPage?: boolean;
+    apiParams?: object;
```

Comment:
```suggestion
    apiParams?: AdditionalDataViewParameters;
```
Shouldn't this be an explicit type of `AdditionalDataViewParameters` instead of just `object`?  In the CustomChecksView.vue we pass an object of type `AdditionalDataViewParameters` to the `DataView` api-params component parameter.

Reply:
I won't need these changes to DataView after https://github.com/Particular/ServicePulse/pull/1793